app/components/embed/media_with_companion_windows_component.html.erb

```diff
@@ -90,9 +90,17 @@
           <div class="header-background">
             <h3>Transcript</h3>
           </div>
-          <div class="autoscroll header-background">
-              <label for="autoscroll">Autoscroll</label>
-              <input data-transcript-target="autoscroll" data-action="transcript#toggleAutoscroll" type="checkbox" class="form-check-input" id="autoscroll" checked>
+          <div class="transcript-controls header-background">
```

Comment:
Can we move all this indside the `.header-background` on line 90?  Then we don't have two header-background containers?